index.html

```diff
@@ -56,6 +56,120 @@ <h4>LET’S CONNECT</h4>
           </li>
         </ul>
       </section>
+      <section id="works">
+        <div id="work-item-one">
+            <img src="images/Snapshoot-Portfolio.png">
+            <h1>Tonic</h1>
+             <ul id="first-list">
+                <li id="canopy">CANOPY</li>
+                <li><img src="images/Counter.png">Back End Dev</li>
+                <li><img src="images/Counter.png"> 2015 </li></ul>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul id="languages">
+                <li>html</li>
+                <li>CSS</li>
+                <li>javaScript</li>
+            </ul>
+            <span><h>See Project</h2></span>
+        </div>
+      </section>
+      <section id="works">
+        <div id="work-item-one">
+            <img src="images/Snapshoot-Portfolio2.png">
+            <h1>Multi-POST STORIES</h1>
+             <ul id="first-list">
+                <li id="canopy">CANOPY</li>
+                <li><img src="images/Counter.png">Back End Dev</li>
+                <li><img src="images/Counter.png"> 2015 </li></ul>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul id="languages">
+                <li>html</li>
+                <li>CSS</li>
+                <li>javaScript</li>
+            </ul>
+            <span><h>See Project</h2></span>
+        </div>
+      </section>
+      <section id="works">
+        <div id="work-item-one">
+            <img src="images/snapshoot-portfolio3.png">
+            <h1>Tonic</h1>
+             <ul id="first-list">
+                <li id="canopy">CANOPY</li>
+                <li><img src="images/Counter.png">Back End Dev</li>
+                <li><img src="images/Counter.png"> 2015 </li></ul>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul id="languages">
+                <li>html</li>
+                <li>CSS</li>
+                <li>javaScript</li>
+            </ul>
+            <span><h>See Project</h2></span>
+        </div>
+      </section>
+      <section id="works">
+        <div id="work-item-one">
+            <img src="images/Snapshoot-Portfolio4.png">
+            <h1>Multi-POST STORIES</h1>
+             <ul id="first-list">
+                <li id="canopy">CANOPY</li>
+                <li><img src="images/Counter.png">Back End Dev</li>
+                <li><img src="images/Counter.png"> 2015 </li></ul>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul id="languages">
+                <li>html</li>
+                <li>CSS</li>
+                <li>javaScript</li>
```

Comment:
[OPTIONAL] Kindly align the items to the left to match the Figma template.
**Your Design**
![alin](https://user-images.githubusercontent.com/73854999/198832416-1329c07a-13c0-4948-8153-563393ab4df6.PNG)

**Figma Design**
![allll](https://user-images.githubusercontent.com/73854999/198832403-bb2238d9-0190-4cce-bfea-86954be17893.PNG)